resources/lib/ui/embed_extractor.py

```diff
@@ -253,7 +268,7 @@ def f(url, content):
 
 __register_extractor("http://embed.novamov.com/", __extract_swf_player)
 
-__register_extractor("https://openload.co/embed/", __extract_openload)
+#__register_extractor("https://openload.co/embed/", __extract_openload)
```

Comment:
also, let's have a TODO note pointing to here:
https://github.com/DxCx/plugin.video.9anime/issues/16